packages/ui/src/CopyInput.tsx

```diff
@@ -21,26 +23,43 @@ export const CopyInput: React.FC<CopyInputProps> = ({
 }) => {
   const { onCopy, hasCopied } = useClipboard(value);
   const theme = useTheme();
+  const [buttonWidth, buttonHeight] = {
```

Comment:
Where do we derive these ratios from?
I noticed in the admin dashboard, that the button overlaps and covers the input border and looks weird. When I adjusted `sm` to have a height of `28px`, the issue went away.
Before:

![Screenshot 2023-09-01 at 11 29 14 AM](https://github.com/fedimint/ui/assets/133234413/f7187e88-a9d8-427f-afb1-3ee339e7966c)

After (`28px` for `sm`):

![Screenshot 2023-09-01 at 11 31 44 AM](https://github.com/fedimint/ui/assets/133234413/1baf8991-328d-4cc4-8a83-9da8233bfc76)

Reply:
What browser is this? It's possible there's some rendering quirks due to how precisely this is placed. Your `28px` has these white gaps around the button that we want to avoid. The goal of these heights is to make it fit within the input's borders, so I think they all net out to `inputHeight - 2px`. However I don't love the way these are done, but it's to deal with Chakra's lack of built-in support for a button inside of an input that's flush with the borders.

If we're willing to deviate from the design, we could place the button flush _beside_ the input, or we can place the button with some padding inset inside the input: https://chakra-ui.com/docs/components/input.

Reply:
I've adjusted the code here to be more dynamic, and was unable to replicate your issue in any browser:

<img width="1890" alt="Screenshot 2023-09-01 at 5 46 52 PM" src="https://github.com/fedimint/ui/assets/649992/be109a99-29dd-46d2-b126-d3338a4f63d2">

LMK if you still see it, maybe you can tinker with the inspector to figure out what's wrong?

Reply:
I no longer see it. Maybe just a glitch in my env.